terraform/messaging.tf

```diff
@@ -144,3 +161,37 @@ resource "aws_sqs_queue_policy" "ingest" {
     }]
   })
 }
+
+resource "aws_sqs_queue" "product_create" {
+  name = "${local.resource_prefix}-product-create-queue"
+  kms_master_key_id = aws_kms_key.product_create.id
+  message_retention_seconds = 7 * 24 * 60 * 60  # 1 week
+  visibility_timeout_seconds = 12 * 60 * 60     # 12 hours
```

Comment:
12 hours seems a bit long for the default visibility timeout shouldn't the message get processed within minutes?

Reply:
It should; I was thinking more of the edge case where the message is hitting an error while being received by the lambdas. Instead of trying to have it retry immediately, it'll be delayed by 12 hours; which hopefully by then the issue resolves. Let me know if that doesn't really make sense here or we want to use a dead-letter queue instead

https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-visibility-timeout.html

Reply:
I think DLQ is the better way to go. We can split that into another ticket though